api/lql_delete.go

```diff
@@ -0,0 +1,52 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import (
+	"net/url"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	lqlDeleteBadInputMsg string = "Please specify a valid query ID."
```

Comment:
These kinds of messages look like are meant to be for the Lacework CLI and not the API client.
All messages you have defined for API client start with a lower case and without a period in the end,
this is one that doesn't follow the pattern but it does follow the pattern we follow for CLI messages.

```suggestion
	lqlDeleteBadInputMsg string = "query ID must be provided"
```

Reply:
Fixed.